.github/workflows/integration-tests.yml

```diff
@@ -1,8 +1,8 @@
 name: Integration
 
 on:
-  push:
-  pull_request:
+  ? push
```

Comment:
this breaks the workflow file

Reply:
Can you please explain how this breaks? Thanks

Reply:
hmm, looking in details in the YAML spec, it seems like this `?` character indicating the beginning of a mapping key is indeed valid. However, it is quite uncommon (and I'm almost sure symfony/yaml fails at parsing it)

Reply:
I can undo that if that is a blocker for this to be merged. But are you sure it is failing? Thanks